systers_portal/meetup/views.py

```diff
@@ -28,6 +29,11 @@
 from common.models import Comment
 from django.http import JsonResponse
 from rest_framework.views import APIView
+from django.views.decorators.csrf import csrf_exempt
+from cities_light.models import City
+from geopy.geocoders import Nominatim
+from django.contrib.gis.geos import Point
```

Comment:
Travis error - `django.core.exceptions.ImproperlyConfigured: Could not find the GDAL library (tried "gdal", "GDAL", "gdal2.1.0", "gdal2.0.0", "gdal1.11.0", "gdal1.10.0", "gdal1.9.0"). Is GDAL installed? If it is, try setting GDAL_LIBRARY_PATH in your settings.`

If GDAL is required, and if you've forgotten to include it, please include it in the requirements file. I suspect this may be the reason Travis is failing and resulting in the error `KeyError: 'en-us'`.